.dotnet/src/Custom/Audio/AudioClient.cs

```diff
@@ -14,7 +14,7 @@ namespace OpenAI.Audio;
 /// <summary> The service client for OpenAI audio operations. </summary>
 [CodeGenClient("Audio")]
 [CodeGenSuppress("AudioClient", typeof(ClientPipeline), typeof(ApiKeyCredential), typeof(Uri))]
-[CodeGenSuppress("CreateSpeechAsync", typeof(SpeechGenerationOptions))]
+[CodeGenSuppress("CreateSpeechAsync", typeof(SpeechGenerationOptions), typeof(CancellationToken))]
```

Comment:
Good catch - both sync and async methods should have a cancellation token. We will fix this in MGC, but it shouldn't impact the generated code (it would just impact the suppression for the sync method once the fix is consumed).

Reply:
Fixed in https://github.com/microsoft/typespec/pull/5337

Reply:
Updated to latest generator version that includes cancellation tokens for sync methods.